### ISSUE TITLE
API silently fails on downloading files with `{ file_type: "zip" }`

### ISSUE THREAD

**Issue by notadamking** [HelloFax/hellosign-nodejs-sdk#66]
The SDK fails silently when requesting to download files for a signature request with:
`{ file_type: "zip" }`

Here is the function I am using to download the files.

```js
  const createDownloadStream = (
    signatureRequestId,
    { file_type = 'pdf' } = { file_type: 'pdf' }
  ) => {
    console.log('Downloading stream');

    return new Promise((resolve, reject) =>
      helloSignClient.signatureRequest.download(
        signatureRequestId,
        { file_type: fileType },
        function callback(err, responseStream) {
          console.log('Downloaded', err, response); // <-- This line is never reached, because the callback is never called.

          if (err) {
            reject(err);
          }

          resolve(responseStream);
        }
      )
    );
  };
```

This has been working for over a year now, and just started failing last week.

**Comment by hazemhamedhs** [comment]
Thank you for reporting that out, Adam.

We filed an internal ticket with the engineering team to tackle this bug [ID-3752] and will keep you posted when the fix is released via ticket #10714791.

**Comment by hazemhamedhs** [comment]
Closing the SDK specific issue for now while the engineering team will work on the internal matter with the HTTP REST API [ID-3752].
